### PR TITLE
Bug 1583640, added CRIO-O port prereq

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1583640

### CODE REVIEW

**Comment by e-minguez** [comment]
LGTM!

**Comment by ahardin-rh** [comment]
@openshift/team-documentation PTAL

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -547,6 +547,10 @@ post back status, to receive tasks, and so on.
 | *10250*
 |TCP
 |The master proxies to node hosts via the Kubelet for `oc` commands.
+
+| *10010*
+|TCP
+|If using CRI-O, open this port to perform `oc exec` and `oc rsh` operations.
```
Comment (mburke5678) [inline install_config/install/prerequisites.adoc]: @ahardin-rh Am I being to picky to suggest s/perform/allow? This almost sound like opening the port does the operations. Or not.....

**Comment by bmeng** [comment]
Doc change looks good, QE reviewed.

**Comment by ahardin-rh** [comment]
@mburke5678 Thank you! Updated. 
Thanks, @bmeng

**Pull request merged**